appendices/clua/bgee.htm

```diff
@@ -202,6 +202,14 @@
         Place mouse cursor of creature to evaluate scripts attached to that creature.<br />
         Otherwise it will evaluate area scripts.<br />
       </div>
+      Eval('RegainRangerHood()')<br />
+      <div class="indent1">
+        Removes the "Fallen Ranger" status from the selected character.<br />
+      </div>
+      Eval('RegainPaladinHood()')<br />
```

Comment:
Eval allows you to run any scripting action, right? You can add a generic example instead of RegainPaladinHood then.